src/api/test/unit/code_quality_test.rb

```diff
@@ -77,7 +77,7 @@ def setup
       'BsRequestActionSubmit#execute_accept'                                    => 126.42,
       'BsRequestPermissionCheck#cmd_changestate_permissions'                    => 117.09,
       'RequestSourceDiff::ActionSourceDiffer#diff_for_source'                   => 94.62,
-      'BuildController#file'                                                    => 127.42,
+      'BuildController#file'                                                    => 130.44,
```

Comment:
I mean `code_quality_test`, it seems like we are just ignoring it entirely - everytime there is a method which has too high complexity we just add it to this blacklist. 

Its not a big deal though. It just seemed kind of weird that when I changed `@http_user` to `User.current` inside that method I also had to update the complexity number in this blacklist in order for the test to pass..